src/App.js

```diff
@@ -1,10 +1,49 @@
 import React, { Component } from 'react'
 import { injectGlobal } from 'emotion'
 import styled from 'react-emotion'
+import Activitylist from './Activitylist'
```

Comment:
Bitte die Datei umbenennen, da du da nur Daten exportierst und keine Component drin hast.

Reply:
Z.B. in `activities`